node/get-anycloud.js

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env node
+
+const { exec, } = require('child_process');
+const path = require('path');
+
+const anycloudUrlBase = 'https://github.com/alantech/anycloud/releases/download/v0.1.1/';
+let extract = '';
+let file = 'anycloud-ubuntu.tar.gz';
+if (process.platform === 'darwin') {
+  file = 'anycloud-macos.tar.gz';
+  extract = `tar -xzf ${file}`;
+} else if (process.platform === 'win32') {
+  file = 'anycloud-windows.zip';
+  extract = 'Expand-Archive -Path anycloud-windows.zip -DestinationPath .';
+} else {
+  extract = `tar -xzf ${file}`;
+}
+
+exec('mkdir bin', (error, stdout, stderr) => {
+  if (error) {
```

Comment:
if it is a `File exists` error should we remove it and get the latest?

Reply:
That should only be a problem for local development on the package and trying to install it, not for the end users who are installing it into their system or app, though, so I don't know if complicating this script is worth it.

Reply:
ah got it, this runs on `preinstall` for the npm package. makes sense